egs/chime4/s5_1ch/local/chime4_train_rnnlms.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
@hainan-xv it is fine to do this review in a week or two. As the PR has already been merged i is not urgent.